NAM/convnet.cpp

```diff
@@ -109,8 +109,13 @@ convnet::ConvNet::ConvNet(const int channels, const std::vector<int>& dilations,
   this->_head = _Head(channels, it);
   if (it != params.end())
     throw std::runtime_error("Didn't touch all the params when initializing ConvNet");
+
+  _prewarm_samples = 1;
+  for (size_t i = 0; i < dilations.size(); i++)
+    _prewarm_samples += dilations[i];
```

Comment:
This should be something like `dilations[i] * (kernel_size[i] - 1)`

_However_, [the kernel size for the ConvNet is hard-coded to 2](https://github.com/sdatkinson/NeuralAmpModelerCore/blob/3152c0e397f52d177bd127e1089f57a39950eb86/NAM/convnet.cpp#L56) so technically this should be correct.

I'm happy to merge this as-is and grab it myself instead of pestering you for a revision.